spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index Page' do
+  before(:each) do
+    @user1 = create(:user, name: 'user1', photo: 'user1_profile_image.jpg', postsCounter: 0)
+    @user2 = create(:user, name: 'user2', photo: 'user2_profile_image.jpg', postsCounter: 1)
+  end
+  scenario 'I can see the username of all other users' do
+    visit users_path
+    expect(page).to have_content(@user1.name)
+    expect(page).to have_content(@user2.name)
+  end
+  scenario 'I can see the profile picture for each user' do
+    visit user_path(@user1)
+    expect(page).to have_css('img[src*="user1_profile_image.jpg"]')
+  end
+  scenario 'I can see the number of posts of the user' do
+    visit user_path(@user1)
+    expect(page).to have_content(@user1.postsCounter)
+  end
+end
```

Comment:
- [ ] As per the requirements, kindly add [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20on%20a%20user%2C%20I%20am%20redirected%20to%20that%20user%27s%20show%20page.) test case. (Kindly note that some of the tests should simulate user interaction using the `capybara` gem, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Use%20Capybara%20to%20write%20integration%20tests%20for%20each%20view%20in%20your%20project.) requirement)